src/world/world.c

```diff
@@ -2,8 +2,23 @@
 #include "ld_addrs.h"
 #include "map.h"
 
+#define ASSET_TABLE_ROM_START 0x1E40000
```

Comment:
This should probably be the linker symbol for the assets segment.

Reply:
He said it wouldn't match to use that since it's not doing addiu

Reply:
Yeah, lui + ori means it was a compile time constant because the mips lo16 and hi16 relocs are not compatible with that pair of instructions.

Reply:
Gotcha, I didn't realise.
Perhaps its worth commenting that at some point as it could possibly confuse people once we get shiftability (or use the linker symbol when defined(NON_MATCHING)). No need to do that in this PR though!